test/statements_tets.cpp

```diff
@@ -189,3 +189,20 @@ R"(
 {
     params = {{"outer", "World Hello"}};
 }
+
+TEST(FilterStatement, General)
```

Comment:
Can you add extra test for applying two (or more) filters with params?

Reply:
done